moves/models.py

```diff
@@ -32,6 +32,7 @@ def save(self, *args, **kwargs):
         if not settings.DEBUG:
             self.post_to_slack()
         super(Move, self).save(*args, **kwargs)
+        self.post_to_slack()
```

Comment:
One of the things that got added into master while this branch is progress was only posting to slack if lunchmove was running in production (that is if `settings.DEBUG` is true. With the merge, it looks like two `self.post_to_slack()` statements got left in which means that it will post to slack twice in production!